Hospital/src/customExceptions/AppointmentListEmpty.java

```diff
@@ -0,0 +1,9 @@
+package customExceptions;
+
+public class AppointmentListEmpty extends Exception {
```

Comment:
add Exception to the exception class name, check all classes
```suggestion
public class AppointmentListEmptyException extends Exception {
```